crates/ruff_python_ast/src/comparable.rs

```diff
@@ -334,7 +334,7 @@ impl<'a> From<&'a ast::Decorator> for ComparableDecorator<'a> {
 pub enum ComparableConstant<'a> {
     None,
     Bool(&'a bool),
-    Str(&'a str),
+    Str { value: &'a str, unicode: bool },
```

Comment:
I added this mainly as it's present in `ExprConstant` for `ComparableExpr` as well but I just checked this in Python that `"foo" == u"foo"` gives `True`. I think then the `unicode` field here should be removed?

Reply:
I'm not sure. Mainly because I never fully understood `ComparableExpr`. Is its intend to test if two trees are identical or if they resolve to the same runtime value? I think the later isn't true because that would require more than just comparing the expression trees but some of the implementation certainly does more so that a isn't true either.

Reply:
That's a good point. I think we should leave it as it is to match the old behavior.